src/scripts/extensions/authenticationHelper.ts

```diff
@@ -154,7 +160,13 @@ export class AuthenticationHelper {
 				// The false case is expected behavior if the user has not signed in or credentials have expired
 				resolve({ parsedResponse: this.isValidUserInformationJsonString(response) ? response : undefined, request: request });
 			}, (error: OneNoteApi.RequestError) => {
+				retrieveUserInformationEvent.setStatus(Log.Status.Failed);
```

Comment:
I like the idea of constancy, but in this case the OneNoteApiUtils is (currently) only available in UI thread.  I'm hesitant to pull this into the extension code unless there is good reason (and maybe even a little bit of refactoring).